src/data/complex/module.lean

```diff
@@ -211,8 +211,8 @@ by rw [← finite_dimensional.finrank_mul_finrank ℝ ℂ E, complex.finrank_rea
 @[priority 900]
 instance star_module.complex_to_real {E : Type*} [add_comm_group E] [has_star E] [module ℂ E]
   [star_module ℂ E] : star_module ℝ E :=
-⟨λ r a, by rw [star_trivial r, restrict_scalars_smul_def, restrict_scalars_smul_def, star_smul,
-  complex.coe_algebra_map, complex.star_def, complex.conj_of_real]⟩
+⟨λ r a, by rw [← one_smul ℂ (star a), ← smul_assoc, ← star_one, ← star_smul, ← star_smul,
+    smul_assoc, one_smul]⟩
```

Comment:
Untested (because my orange bars restarted before I could finish the rewrite chain),
```suggestion
⟨λ r a, by rw [←smul_one_smul ℂ r a, star_smul, star_smul, star_one, smul_one_smul]⟩
```

Reply:
Or even
```suggestion
⟨λ r a, by simp_rw [←smul_one_smul ℂ (_ : ℝ) (_ : E), star_smul, star_one]⟩
```

Reply:
I think I was clicking commit as the second version landed, though I can confirm both work.

I think I'll just wait for CI now and then send to bors.